From: Robert Milasan <rmilasan@suse.com>
Date: Tue, 12 Feb 2013 09:16:23 +0000
Subject: create default links for primary cd_dvd drive

cdrom_id: created links for the default cd/dvd drive (bnc#783054).
---
 rules/60-cdrom_id.rules | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff -Naur systemd-244.20191203/rules.d/60-cdrom_id.rules systemd-244.20191203.tpg/rules.d/60-cdrom_id.rules
--- systemd-244.20191203/rules.d/60-cdrom_id.rules	2019-11-29 16:32:36.000000000 +0000
+++ systemd-244.20191203.tpg/rules.d/60-cdrom_id.rules	2019-12-03 12:16:25.896021820 +0000
@@ -24,6 +24,9 @@
 # stale mounts after ejecting
 ENV{DISK_MEDIA_CHANGE}=="?*", ENV{ID_CDROM_MEDIA}!="?*", ENV{SYSTEMD_READY}="0"
 
-KERNEL=="sr0", SYMLINK+="cdrom", OPTIONS+="link_priority=-100"
+KERNEL=="sr0", ENV{ID_CDROM}=="1", SYMLINK+="cdrom", OPTIONS+="link_priority=-100"
+KERNEL=="sr0", ENV{ID_CDROM_CD_RW}=="1", SYMLINK+="cdrw", OPTIONS+="link_priority=-100"
+KERNEL=="sr0", ENV{ID_CDROM_DVD}=="1", SYMLINK+="dvd", OPTIONS+="link_priority=-100"
+KERNEL=="sr0", ENV{ID_CDROM_DVD_RW}=="1", SYMLINK+="dvdrw", OPTIONS+="link_priority=-100"
 
 LABEL="cdrom_end"
